spec/features/solidus_importer/import_spec.rb

```diff
@@ -30,9 +29,9 @@
     let(:import_type) { :products }
     let(:csv_file_rows) { 7 }
     let(:product_slug) { 'hightop-sports-sneaker' }
-    let!(:shipping_category) { create(:shipping_category) }
+    let(:shipping_category) { create(:shipping_category) }
 
-    after { shipping_category.destroy }
+    before { shipping_category }
```

Comment:
This looks like a typo, did you mean to replace it with a `let!`?

Reply:
It was in that way because of `RSpec/LetSetup` Rubocop's rule.
But thinking about it I prefer to disable that rule that could bring more confusion than benefits.